packages/utils/src/lib/reports/scoring.unit.test.ts

```diff
@@ -52,23 +52,52 @@ describe('calculateScore', () => {
       ),
     ).toBeCloseTo(0.92);
   });
+});
 
+describe('parseScoringParameters', () => {
```

Comment:
Maybe better to test these for the `calculateScore` function instead. That's the one we care about, whereas `parseScoringParameters` can be an internal implementation detail (doesn't need to be exported at all).

Reply:
Good point, fixed